docs/tuning.rst

```diff
@@ -350,6 +386,14 @@ minimal set of torrents you are actually interested in.
 To get an update with only the torrents that have changed since last time, call
 ``session::post_torrent_updates()``.
 
+high latency networks
+=====================
+
```

Comment:
I'm not sure about this. It's a very interesting topic but are there any fast networks with high latency out there? I know only networks with extrem proxy setups for privacy and I don't know any non suspicious usecases for large file transfere in these kind of networks.